tests/constants/githublabel.constants.ts

```diff
@@ -2,56 +2,40 @@ import { GithubLabel } from '../../src/app/core/models/github/github-label.model
 
 export const GITHUB_LABEL_FUNCTIONALITY_BUG = {
   color: '9900cc',
-  name: 'type.FunctionalityBug',
-  id: 1226647550,
-  url: 'https://api.github.com/repos/CATcher-org/pe-results/labels/type.FunctionalityBug'
+  name: 'type.FunctionalityBug'
 } as GithubLabel;
 
 export const GITHUB_LABEL_FEATURE_FLAW = {
-  id: 1462485994,
-  url: 'https://api.github.com/repos/CATcher-org/pe-results/labels/type.FeatureFlaw',
   name: 'type.FeatureFlaw',
```

Comment:
Let's take the opportunity to standardise the property order for all constants to `color` then `name`, according to the order in `github-label.model.ts`